tests/qos/tunnel_qos_remap_base.py

```diff
@@ -95,8 +95,8 @@ def get_queue_counter(duthost, port, queue, clear_before_read=False):
     """
     if clear_before_read:
         duthost.shell("sonic-clear queuecounters")
-    # Wait a default interval (10 seconds)
-    time.sleep(10)
+        # Wait a default interval (10 seconds)
```

Comment:
So this is caused by the formatting alignment?

Reply:
Not sure of the original purpose, but it leads to checking bounced traffic and queue counters happening after the pfc pause frame is sent